src/DependencyInjection/AceEditorExtension.php

```diff
@@ -42,4 +44,35 @@ private function registerAceEditorParameters(array $config, ContainerBuilder $co
         $container->setParameter('ace_editor.options.base_path', $config['base_path']);
         $container->setParameter('ace_editor.options.mode', $mode);
     }
+
+	/**
+	 * @see https://symfony.com/doc/current/frontend/create_ux_bundle.html
+	 */
+    public function prepend(ContainerBuilder $container)
+    {
+        if ($this->isAssetMapperAvailable($container)) {
+            $container->prependExtensionConfig('framework', [
+                'asset_mapper' => [
+                    'paths' => [
+                        __DIR__ . '/../../assets/controllers' => 'norberttech/aceeditor-bundle',
```

Comment:
some time ago we renamed namespace from NorbertTech into AceEditorBundle, I think this path should reflect it as well

Reply:
No, this is not namespace from code but name from composer.json. need to be same